openstack/scheduler/templates/domain-ccadmin-seed.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: "OpenstackSeed"
+metadata:
+  name: scheduler-seed
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"

```

Comment:
Same with this stuff. Helm automatically adds annotations to track provenance of the objects that it provisions, so those can be used to trace back the seed to its containing Helm release.